crossterm_terminal/src/terminal/winapi_terminal.rs

```diff
@@ -226,7 +226,7 @@ pub fn clear_entire_screen(buffer_size: Size, current_attribute: u16) -> Result<
     clear(start_location, cells_to_write, current_attribute)?;
 
     // put the cursor back at cell 0,0
-    let cursor = Cursor::new()?;
+    let cursor = TerminalCursor::new();
```

Comment:
We can use `ScreenBufferCursor` directly because we are using winapi here and thus there is no need to go through the abstraction of ansi/winapi

Reply:
Disagree here as it's an implementation detail of the `crossterm_cursor` crate (private `sys` module).